### PR TITLE
Acceptance tests

### PR DESCRIPTION
**Pull request by shin-**
This PR adds the ability to run the [acceptance test suite](https://github.com/docker/compose/tree/master/tests/acceptance) from docker/compose against the libcompose CLI. It is configured to run with the `test` make target (or individually with `test-acceptance`).

cc @dnephin

### CODE REVIEW

File: script/test-acceptance [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+set -e
+
+export SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+export DEST=.
+
+bundle .integration-daemon-start
+
+apt-get install -y python-setuptools
+easy_install pip
+pip install virtualenv
```
Comment (vdemeester) [inline script/test-acceptance]: @shin- I would definitely put these steps in the `Dockerfile` and not there

- I don't want it to run each time I run `script/test-acceptance`
- I want to be able to run this script (outside the docker container, why not :stuck_out_tongue_closed_eyes:) on my Archlinux for example (but could be on a fedora, …) where, I don't have `apt-get`

These (`setuptools`, `pip` and `virtualenv`) are requirements to execute this test-acceptance, but they definitely shouldn't be in the script.

**Comment by vdemeester** [comment]
@shin- Very cool :raised_hands: I Added a remark on the script

How do we handle the "missing" part (stuff that do not work or not yet implemented) for now ? There is 2 options I see : 
- Waiting for them to get fixed (and rebasing / using this PR/branch to validate this) and then merge
- Merge this one but make the build success if the acceptance test fails ; and when it's all *conform*, we can re-enable *strict* checking on the acceptance test.

@docker/libcompose-maintainers wdyt ?

PS : I'm preparing an issue to keep track of the missing part libcompose has with docker-compose 1.5 so we can focus on working on these :wink:.

File: script/test-acceptance (hunk 1 above)
Comment (dnephin) [inline script/test-acceptance]: sounds right to me

**Comment by aanand** [comment]
I think I'm leaning toward making it a non-critical part of the test suite, so that it doesn't hold up work on anything else. It seems about the right level of annoying to have it produce a lot of failing output, but not actually break the build.

**Comment by vdemeester** [comment]
I didn't answer my own question, I'm also for a non-critical part for now :)

**Comment by shin-** [comment]
Updated addressing (hopefully) everyone's comments. Thank you!

**Comment by dnephin** [comment]
LGTM!

**Comment by vdemeester** [comment]
@shin- could you squash your 4 last commits ? and it will definitely LGTM for me :stuck_out_tongue_closed_eyes:

**Comment by shin-** [comment]
Squashed :+1:

**Comment by aanand** [comment]
Nice one.

**Pull request merged**